### PR TITLE
Sc 41611/rename trustedform consent data integration

### PR DESCRIPTION
**Pull request by johnrb2**
## Description of the change

> Found references to Consent + Data and renamed data to insights.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change
- [ ] Technical Debt
- [ ] Documentation

## Related tickets

> https://app.shortcut.com/active-prospect/story/41611/rename-trustedform-consent-data-integration-to-trustedform-consent-insights

### Development and Testing

- [x]  Lint rules pass locally.
- [x]  The code changed/added as part of this pull request has been covered with tests, or this PR does not alter production code.
- [x]  All tests related to the changed code pass in development, or tests are not applicable.

### Code Review

- [x]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.
- [x]  At least two engineers have been added as "Reviewers" on the pull request.
- [x]  Changes have been reviewed by at least two other engineers who did not write the code.
- [x]  This branch has been rebased off master to be current.

### Tracking 
- [x]  Issue from Shortcut/Jira has a link to this pull request.
- [x]  This PR has a link to the issue in Shortcut.

### QA
- [ ]  This branch has been deployed to staging and tested.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ module.exports = {
   outbound: {
     claim: require('./lib/claim'),
     consent: require('./lib/consent'),
-    consent_plus_data: require('./lib/consent_plus_data'),
-    data_service: require('./lib/data_service'),
-    decision_service_ping: require('./lib/data_service'),
-    decision_service_post: require('./lib/data_service')
+    consent_plus_insights: require('./lib/consent_plus_insights'),
+    insights_service: require('./lib/insights_service'),
```
Comment (cgrayson) [inline index.js:7]: I think the name will just be "insights", not "insights service" (the same way that "consent plus data" isn't changing to "consent plus insight service").

File: lib/insights_service.js [hunk 2]
```diff
@@ -131,40 +131,40 @@ const response = (vars, req, res) => {
...
-  { name: 'data_service.user_agent', type: 'string', description: 'The browser and operating system for the device on which the consumer filled out the form' },
-  { name: 'data_service.fingerprints_matching', type: 'array', description: 'Matching fingerprints that were found on the TrustedForm certificate' },
-  { name: 'data_service.fingerprints_non_matching', type: 'array', description: 'Non-matching fingerprints that were found on the TrustedForm certificate' },
-  { name: 'data_service.is_masked', type: 'boolean', description: 'Whether or not the TrustedForm certificate is masked' },
-  { name: 'data_service.scans_found', type: 'array', description: 'Language that was found through the use of page scanning' },
-  { name: 'data_service.scans_not_found', type: 'array', description: 'Language that was not found through the use of page scanning' },
-  { name: 'data_service.warnings', type: 'array', description: 'Any warnings present about the content of the form, specific to page scanning and fingerprinting' }
+  { name: 'insights_service.outcome', type: 'string', description: 'Integration outcome (success, failure, or error)' },
```
Comment (cgrayson) [inline lib/insights_service.js:134]: I think this will change, but there's still an outstanding question as to what exactly it should change to.

**Review by cgrayson — DISMISSED** [review]
Looks like a good start, a couple points inline. Also it looks like this covers sc-38895, too, so (1) it will need a migration and (2) tell @richardfordjunior that his work is already done 😉

**Comment by johnrb2** [comment]
Reseting to master, made changes outside the scope of the ticket.

**Comment by johnrb2** [comment]
Re-openning

**Review by jgrayson676 — APPROVED** [review]

**Pull request merged**